homeassistant/components/simplisafe/__init__.py

```diff
@@ -49,9 +49,13 @@
 
 @callback
 def _async_save_refresh_token(hass, config_entry, token):
-    config_entry.data[CONF_TOKEN] = token
     hass.config_entries.async_update_entry(
-        config_entry, data=config_entry.data)
+        config_entry,
+        data={
```

Comment:
This also works and is less likely to break in the future: `{**config_entry.data, CONF_TOKEN: token}`

Reply:
http://treyhunner.com/2018/10/asterisks-in-python-what-they-are-and-how-to-use-them/